include/wx/qt/textentry.h

```diff
@@ -36,13 +38,15 @@ class WXDLLIMPEXP_CORE wxTextEntry : public wxTextEntryBase
     virtual bool IsEditable() const;
     virtual void SetEditable(bool editable);
     
+    virtual void SetMaxLength(unsigned long len);
 protected:
     virtual wxString DoGetValue() const;
     virtual void DoSetValue(const wxString& value, int flags=0);
 
     virtual wxWindow *GetEditableWindow();
 
 private:
+    virtual QLineEdit *GetEditable()  const { return NULL; };
```

Comment:
It doesn't make sense to call this function `GetEditable()`, it's called like this in wxGTK because it returns `GtkEditable`, but here it really should be called `GetLineEdit()` or maybe `QTGetLineEdit()`.

Also, shouldn't it be pure virtual?